xml/common_intro_available_doc.xml

```diff
@@ -45,7 +45,7 @@
    <!--based on 'Support Resources' listed at https://www.suse.com/support/kb/-->
    <listitem>
     <para>
-     If you have run into an issue, also check out the Technical Information
```

Comment:
This needs to be fixed in doc-kit actually (because this file will be overwritten with the next doc-kit run). I'll fix it in doc-kit, but we can keep it here, too.